src/components/map/map-triangulated-strain-layer.tsx

```diff
@@ -50,53 +63,50 @@ export class MapTriangulatedStrainLayer extends BaseComponent<IProps, IState> {
       const ranges = isLog ? logarithmicStrainRanges : equalIntervalStrainRanges;
 
       const range = ranges.slice().reverse().find(r => strain > r.min);
-      let fillColor;
-      if (range) {
-        fillColor = range.color;
-      } else {
-        fillColor = "#000";
+      let strokeColor = "#333";
+      let fillColor = "none";
+      if (renderStrainMap) {
+        strokeColor = "#FFF";
+        if (range) {
+          fillColor = range.color;
+        } else {
+          fillColor = "#000";
+        }
       }
 
       triangles.push(<Polygon
         positions={[p1, p2, p3]} key={`triangle-${i}`}
         stroke={true}
-        color="#FFF"
+        color={strokeColor}
         weight={1}
         fillOpacity={0.8}
         fillColor={fillColor}
       />);
 
-      // triangles.push(Leaflet.polygon(
-      //   [p1, p2, p3],
-        // {
-        //   stroke: true,
-        //   color: "#FFF",
-        //   weight: 1,
-        //   fillOpacity: 0.8,
-        //   fillColor: this.gradient.rgbAt(adjustedStrainValues[(i - i % 3) / 3]).toRgbString()
-        // }
-      //   ));
-
-      // // calculate the "incenter" of the triangle
-      // const perim1 = Math.sqrt(((p1.lat - p2.lat) ** 2) + ((p1.lng - p2.lng) ** 2));
-      // const perim2 = Math.sqrt(((p2.lat - p3.lat) ** 2) + ((p2.lng - p3.lng) ** 2));
-      // const perim3 = Math.sqrt(((p3.lat - p1.lat) ** 2) + ((p3.lng - p1.lng) ** 2));
-      // const perimiter = perim1 + perim2 + perim3;
-      // const centLat = ((p1.lat * perim2) + (p2.lat * perim3) + p3.lat * perim1) / perimiter;
-      // const centLng = ((p1.lng * perim2) + (p2.lng * perim3) + p3.lng * perim1) / perimiter;
-      // const incenter = Leaflet.latLng(centLat, centLng);
-
-      // // Leaflet.divIcon({iconAnchor: incenter})
-      // Leaflet.circle(incenter, {radius: 1})
-      //   .bindTooltip("" + (Math.round(adjustedStrainValues[(i - i % 3) / 3] * 10000) / 10000), {
-      //     permanent: true,
-      //     className: "plain-label"
-      //   })
-      //   .addTo(map);
+      // calculate the incenter of the triangle
+      const perim1 = Math.sqrt(((p1.lat - p2.lat) ** 2) + ((p1.lng - p2.lng) ** 2));
+      const perim2 = Math.sqrt(((p2.lat - p3.lat) ** 2) + ((p2.lng - p3.lng) ** 2));
+      const perim3 = Math.sqrt(((p3.lat - p1.lat) ** 2) + ((p3.lng - p1.lng) ** 2));
+      const perimeter = perim1 + perim2 + perim3;
+      const centLat = ((p1.lat * perim2) + (p2.lat * perim3) + p3.lat * perim1) / perimeter;
+      const centLng = ((p1.lng * perim2) + (p2.lng * perim3) + p3.lng * perim1) / perimeter;
+      const incenter = Leaflet.latLng(centLat, centLng);
+      const minPerimSize = 1.5 - ((zoomLevel - 6) * 0.4);
+      const largeSide = zoomLevel > 7 ? perimeter * 0.48 : perimeter * 0.45;
+      const smallSide = zoomLevel > 7 ? perimeter * 0.1 : perimeter * 0.25;
+      console.log(minPerimSize);
```

Comment:
I'm not 100% sure on the stance of leaving console.logs in when merging to master for this codebase. But I thought I'd point this out.

Reply:
:+1: